app/component/itinerary/ItineraryPage.js

```diff
@@ -1096,14 +1111,22 @@ export default function ItineraryPage(props, context) {
   } else if (detailView) {
     if (naviMode) {
       content = (
-        <NaviContainer
-          itinerary={combinedEdges[selectedIndex]?.node}
-          focusToLeg={focusToLeg}
-          relayEnvironment={props.relayEnvironment}
-          combinedEdges={combinedEdges}
-          setNavigation={setNavigation}
-          mapRef={mwtRef.current}
-        />
+        <>
+          <NavigatorIntroModal
```

Comment:
I wonder if rendering should be conditional: {!isNavigatorIntroDismissed && (<NavigatorIntroModal ...
Now rendering loads some resoureces even when they are not shown. IsOpen prop is then unnecessary.

Reply:
Good suggestion. I'll double check to make sure that it wasn't the modal that enforced this sort of implementation and then refactor if need be.